File: programmers/Lv_1/KimHyunseo/문자열내p와y의개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
	bool answer = true;
	int pnum = 0;
	int ynum = 0;
	for (int i = 0; i < s.length(); i++)
	{
		if (s[i] == 'p' || s[i] == 'P')
			pnum ++;
		if (s[i] == 'y' || s[i] == 'Y')
			ynum++;
	}
	if (ynum == pnum)
		answer = true;
	else
		answer = false;
	return answer;
}

int main()
{
	string s = "pPoooyY";
	cout<<solution(s);
}

File: programmers/Lv_1/JeongJunKwon/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int pNum = 0;
        int yNum = 0;
        for(int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if(c == 'p' || c == 'P') {
                pNum++;
            } else if (c == 'y' || c == 'Y') {
                yNum++;
            }
        }
        if(pNum == yNum || (pNum == 0 && yNum == 0))
           return true;
        else
           return false;
    }
}